### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by nonstdio**

### CODE REVIEW

**Comment by dchapmanfw** [comment]
👀

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# FROM: https://raw.githubusercontent.com/github/gitignore/main/Python.gitignore
```
Comment (dchapmanfw) [inline .gitignore:1]: is this a generic git ignore you throw on everything?

File: functions.py [hunk 2]
```diff
@@ -2,17 +2,17 @@
```
Comment (dchapmanfw) [inline functions.py:2]: is there a better name for this file than functions? I guess it kind of is a trash bin of random stuff.

File: functions.py [hunk 3]
```diff
@@ -2,17 +2,17 @@
 
 
 # whenever a user bets it sends this text I just added the semantics part just in case someone already had bet
-def userInputPts(user, amount, blvPercent, dbtPercent, side, globalDict, believePool, doubtPool):
-    text = f"{user} has added to the pool with **{amount} points! on \"{globalDict[side]}\"** <:vhard:613718800298147890> \n" \
+def user_input_points(user, amount, believe_percent, doubt_percent, side, global_dict, believe_pool, doubt_pool):
```
Comment (dchapmanfw) [inline functions.py:5]: I love seeing a global dictionary being passed as an input to a function.

File: functions.py [hunk 4]
```diff
@@ -23,90 +23,72 @@ def startText(title, blv, dbt, timer):
...
+def reset_after_win(global_dict, believe_pool, doubt_pool, payout_pool):
+    global_dict.clear()
+    believe_pool.clear()
+    doubt_pool.clear()
+    payout_pool.clear()
 
 
 # shows title result percentages, biggest payout
-def returnWinText(title, Result, blvPercent, dbtPercent, side, believePool, doubtPool, payOutPool):
+def return_win_text(title, result, believe_percent, doubt_percent, side, believe_pool, doubt_pool, payout_pool):
```
Comment (dchapmanfw) [inline functions.py:55]: ditto for passing result.. Ok why did I start actually looking at this code.

File: main.py [hunk 5]
```diff
@@ -81,53 +81,54 @@ def listGuild():
...
-                    userPoints = functions.showPoints(collection.find({"name": person.name})) + points
-                    collection.update_one({"name": person.name}, {"$set": {"points": userPoints}})
+                    user_points = functions.show_points(collection.find({"name": person.name})) + points
+                    collection.update_one({"name": person.name}, {"$set": {"points": user_points}})
             else:
                 pass
-        vcList = []
+        voice_channels = []
     # restarts the function every 30 min
-    this = Timer(1800, voiceChannelCheck)
```
Comment (dchapmanfw) [inline main.py:101]: what the actual fuck?

File: main.py [hunk 6]
```diff
@@ -198,29 +200,29 @@ class Predictions(commands.Cog):
...
-            bot.Timer -= 1
-            await message.edit(content=functions.startText(title, blv, dbt, timer))
-        await ctx.invoke(self.bot.get_command('closeSubmissions'))
+            bot.timer -= 1
+            await message.edit(content=functions.start_text(title, blv, dbt, timer))
+        await ctx.invoke(self.bot.get_command('close_submissions'))
 
     @start.error
-    async def start_error(self, ctx, error):
+    async def start_error(self, ctx, error):  # just take it out behind the shed
```
Comment (dchapmanfw) [inline main.py:225]: 😆

File: main.py [hunk 7]
```diff
@@ -229,106 +231,107 @@ async def start_error(self, ctx, error):
...
     @commands.command(aliases=['believe', 'blv'])
-    async def betBelieve(self, ctx, amount: int):
-        user, userMention, thisTime = ctx.message.author.name, ctx.message.author.mention, timeCheck()
-        if isinstance(thisTime, bool):
-            userDB = bot.betCollection.find({"name": user})
-            userPoints = functions.showPoints(userDB)
-            userPoints -= amount
-            if user in doubtPool:
-                text = f"You've chosen your side already {userMention} <:derp:953081066565038130>"
+    async def bet_believe(self, ctx, amount: int):  # there's no hope
```
Comment (dchapmanfw) [inline main.py:234]: I can tell you have been hurt.

File: main.py [hunk 8]
```diff
@@ -229,106 +231,107 @@ async def start_error(self, ctx, error):
...
     @commands.command(aliases=['doubt', 'dbt'])
-    async def betDoubt(self, ctx, amount: int):
-        user, userMention, thisTime = ctx.message.author.name, ctx.message.author.mention, timeCheck()
-        if isinstance(thisTime, bool):
-            userDB = bot.betCollection.find({"name": user})
-            userPoints = functions.showPoints(userDB)
-            userPoints -= amount
-            if user in believePool:
-                text = f"You've chosen your side already {userMention} <:derp:953081066565038130>"
+    async def bet_doubt(self, ctx, amount: int):  # why did I even start?
```
Comment (dchapmanfw) [inline main.py:276]: 😢 It's ok Ben, it's ok

**Review by dchapmanfw — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (nonstdio) [reply .gitignore:1]: github puts this out, they have boilerplate for several languages

File: functions.py (hunk 2 above)
Comment (nonstdio) [reply functions.py:2]: if we decompose it, certainly. the new names will be obvious.

**Pull request closed without merging**